tools/pd-api-bench/README.md

```diff
@@ -0,0 +1,13 @@
+When test ScanRegions and GetRegion, it's better to set region-num.
+When test ScanRegions, it's better to set regions-sample, and not greater than 10000.
+``` bash
+go run ./main.go --region 10 --region-num 200000 --concurrency 10
+go run ./main.go --regions 10 --region-num 200000 --regions-sample 1000 --concurrency 10
```

Comment:
What is the difference among `--regions`, `--region-num`, and `--regions-sample`?

Reply:
`ScanRegions` QPS, the number for total regions, the number of the result regions for a `ScanRegions`
I will modidy param name